tests/nevermind_test.py

```diff
@@ -7,3 +7,5 @@ def test_nevermind(testbot):
     testbot.assertCommand("!nm", "I'm sorry :(")
     testbot.assertCommand("!nEverMINd", "I'm sorry :(")
     testbot.assertCommand("!nM", "I'm sorry :(")
+    testbot.assertCommand("!nm COBOT_PREFIX invite meet",
+                                     "I'm sorry :(")
```

Comment:
The tests don't seem right.. also the tests should go in `labhub_test`.. test this command `!hey there invite me` there should be no response and a timeout error should be raised. See the test files for an example of how its done.

Reply:
I have updated the tests